sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/IntervalUtils.scala

```diff
@@ -482,13 +409,17 @@ object IntervalUtils {
       }
     }
 
+    def nextWord: UTF8String = {
+      s.substring(i, s.numBytes()).subStringIndex(UTF8String.blankString(1), 1)
```

Comment:
This is for error reporting, so perf doesn't matter. Can we make it more readable? like
```
s.substring(i, s.numBytes()).toString.split("\\s+").head
```